Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -15,6 +15,7 @@
 use Doctrine\ORM\QueryBuilder;
 use Dunglas\ApiBundle\Api\IriConverterInterface;
 use Dunglas\ApiBundle\Api\ResourceInterface;
+use InvalidArgumentException;
```

Comment:
The FQCN is `Dunglas\ApiBundle\Exception\InvalidArgumentException` to use our "domain exception".